Changelog.md

```diff
@@ -4,6 +4,16 @@
 Breaking Changes:
 
 * Ruby < 2.3 is no longer supported. (Phil Pirozhkov, #1231)
+* Remove `should` and `should_not` syntax (including one-liners). (Phil Pirozhkov, #1245)
+* Turn `strict_predicate_matchers` on by default. (Phil Pirozhkov, #1277)
+* Remove deprecated `LegacyMacherAdapter`. (Phil Pirozhkov, #1253)
+* Remove support for legacy RSpec matchers (pre 3). (Phil Pirozhkov, #1253)
+* Remove `include_chain_clauses_in_custom_matcher_descriptions` option
+  and make it the default. (Phil Pirozhkov, #1279)
+* Restrict block matchers use with value expectations. (Phil Pirozhkov, #1285)
```

Comment:
I don't get it. This is already in `4-0-dev` :confused: 
https://github.com/rspec/rspec-expectations/blame/4-0-dev/Changelog.md#L6